api/SpeechSynthesisEvent.json

```diff
@@ -238,10 +238,63 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "milliseconds": {
+          "__compat": {
+            "description": "<code>elapsedTime</code> in milliseconds",
+            "support": {
+              "chrome": {
+                "version_added": "33"
+              },
+              "chrome_android": {
+                "version_added": "33"
+              },
+              "edge": {
+                "version_added": "14"
+              },
```

Comment:
@makotokato @ddbeck Thanks very much. 

@ddbeck I couldn't understand your suggestion in https://github.com/mdn/browser-compat-data/pull/13236/files#r743560481

But adding the information seemed pretty clear:
1. For FF desktop we support milliseconds from 49 to 95 partial, in Windows and macOS only. For Linux the implication is that the subfeature doesn't apply.
2. Firefox android only supports seconds forever. So for the milliseconds subfeature the support statement is just `false`.

Is that what you meant?

Reply:
Yes. I misspoke in the original comment. This looks right. 👍